### ISSUE TITLE
Rails 5 Turbolinks 5, not loading after a few page loads

### ISSUE THREAD

**Issue by DLawla** [luizpicolo/tawk-rails#6]
Hi guys,
I'm seeing an issue on Rails 5.1 and Turbolinks 5.0.1 where the Tawk dialog loads correctly, then after approximately 6-7 turbolinks page loads, it stops loading. When I refresh the page, however, it resumes working for the next 6-7 page loads.
Cheers!

**Comment by ArkGladios** [comment]
Same here!

**Comment by dokuzbir** [comment]
still no fix?

**Comment by alex-kovshovik** [comment]
Yep, just tried the latest version, still getting this issue:

![vin deals negotiate a car deal like a pro 2018-03-07 07-43-50](https://user-images.githubusercontent.com/889219/37098535-51461a9e-21db-11e8-8b1a-8c21615094ab.jpg)

> TypeError: this.getBrowserData is not a function

> TypeError: document.body is null

**Comment by dokuzbir** [comment]
Are you sure moving in to body solves bug? It didnt work on my app. Problem seems about network. After 5 page loads tawk_to doesnt dowload some files

**Comment by alex-kovshovik** [comment]
I just tested clicking through my project, like 15 times: it works 100%. I did test both in development mode and in production. I just put `<%= tawk_init %>` as a first statement after `<body>` tag in my `app/layouts/application.html.erb`.

**Comment by mapreal19** [comment]
@alex-kovshovik Unfortunately I've tried in mine as you suggested and after 5-6 times it stopped working. Same as @dokuzbir

**Comment by mapreal19** [comment]
Maybe we should upvote this one: https://tawk.uservoice.com/forums/228427-feedback/suggestions/32883448-add-support-for-rails-turbolinks

**Comment by alex-kovshovik** [comment]
Makes sense, I just cast all 3 of my votes for that feature request @mapreal19: https://tawk.uservoice.com/forums/228427-feedback/suggestions/32883448-add-support-for-rails-turbolinks.

Sorry to hear it doesn't work for you. For some reason, it works for me just fine and my setup is not special at all.

Is it possible that this works for me because I'm using the latest version of Turbolinks (5.0.1)?

**Comment by luizpicolo** [comment]
I did a simple test with two links and it worked correctly.

turbolinks (5.1.1, 5.1.0)
turbolinks-source (5.1.0)
